glyphs-reader/src/font.rs

```diff
@@ -1256,10 +1258,48 @@ impl RawFont {
                 // this is quadratic but N is small
                 if let Some(metric) = metric_values.iter_mut().find(|x| x.pos == Some(pos)) {
                     metric.over = Some(over);
+                } else {
+                    non_metric_alignment_zones[i].push((pos, over))
                 }
             }
             master.metric_values = metric_values;
         }
+
+        // now handle any non-metric alignment zones, converting them to metrics.
+        // first we assign a name to each unique position:
+        let mut new_metrics = HashMap::new();
+        for pos in non_metric_alignment_zones
+            .iter()
+            .flat_map(|master| master.iter().map(|(pos, _)| *pos))
+        {
+            if !new_metrics.contains_key(&pos) {
+                let next_zone = new_metrics.len() + 1;
+                let idx = self.metrics.len();
+                self.metrics.push(RawMetric {
+                    type_: Some(format!("zone {next_zone}")),
```

Comment:
If we’re going to retain these, I’d suggest capturing them into `blue_values` and `other_blues` fields as glyphslib does. I don’t think storing them as “zone _n_” has any useful meaning.

Reply:
as best I understand things that would happen during during the glyphs2fontir stage? All I'm trying to do here is convert v2 to v3, so that when we read glyphs sources we have a single normalized representation at the end.

Reply:
Ah, okay. That makes sense then. I’ll admit to not being familiar with the differences between these formats.